examples/build.gradle

```diff
@@ -90,7 +90,9 @@ dependencies {
     implementation project(':libnavigation-util')
     implementation project(':liblogger')
     implementation project(':libnavigator')
-    androidTestImplementation project(':libtesting')
+    androidTestImplementation(project(':libtesting')) {
+        exclude group: 'org.robolectric'
```

Comment:
Excluded because of 
https://github.com/robolectric/robolectric/issues/5235

Reply:
in what modules do we have duplicates?
can It be fixed in `libtesting` module to avoid such fixes in every consumer module?

Reply:
It's not the only problem.  
The second one - we got 'java.lang.ClassCastException: android.os.Looper cannot be cast to org.robolectric.internal.bytecode.ShadowedObject' when run UI tests on simulators/devices.  Don't know why runner is trying to make shadows of real classes

Reply:
Alternatively we can create different configurations inside `libtesting` module and provide different  dependencies

Reply:
I'm confused why we didn't have such problems before(`duplicating` and `ClassCastException`)? There were no such problems in `driver` repo
Any ideas?

Reply:
We didn't include `robolectric` in `androidTestImplementation` dependencies

Reply:
Don't we? 👀 https://github.com/mapbox/mapbox-navigation-android/blob/f223bc9d6dd33d6aaf622c512ccb35f454b441a2/libtesting/build.gradle#L34

Reply:
@Guardiola31337  The libtesting module was added to `testImplementation`, not `androidTestImplementation` in other modules